lib/codecs/tests/avro.rs

```diff
@@ -0,0 +1,88 @@
+use bytes::{Bytes, BytesMut};
+use codecs::{
+    decoding::format::AvroDeserializerConfig, decoding::format::Deserializer,
+    encoding::format::AvroSerializerConfig,
+};
+use rstest::*;
+use similar_asserts::assert_eq;
+use std::{
+    fs::File,
+    io::Read,
+    path::{Path, PathBuf},
+    str::from_utf8,
+};
+use tokio_util::codec::Encoder;
+use vector_core::{config::LogNamespace, event::Event};
+
+#[rstest]
+fn roundtrip_avro_fixtures2(
+    #[files("tests/data/avro/avro/*.avro")]
+    #[exclude(
+        ".*(date|decimal_var|duration|fixed|map|time_millis|timestamp_micros|timestamp_millis).avro"
+    )]
+    path: PathBuf,
+) {
+    let schema_path = path.as_path().with_extension("avsc");
+    assert!(schema_path.exists());
+
+    roundtrip_avro(path, schema_path, false);
```

Comment:
🥜 Since we are using `rstest`, can this boolean be a `#[case()]` and then merge `roundtrip_avro_fixtures2` and `roundtrip_avro_fixtures_reverse2`?